requirements.txt

```diff
@@ -1,3 +1,4 @@
+PyYAML
```

Comment:
Does this not need to be in setup.py?

Reply:
Neither PyCharm nor Travis has complained..
I've just checked - it's the same case for PySFTP - it's in the requirement.txt file but not the setup.py
I can add them both if you'd like? I'm not too familiar with this aspect of the code

Reply:
Yes could you please add them both to `setup.py`
In theory hard dependencies on the project (e.g. Autoreduction will not work without these) are in `setup.py`. soft dependencies live in `requirements.txt` (e.g. packages that help with testing the project like Mock)

Reply:
Sure, I'll add both to `setup.py`.
Would you like me to remove them from `requirements.txt` too? Or leave them there?

Reply:
Yes can you remove this from `requirements.txt`

Reply:
I've done so for this and #483, but the Travis build seems to fail as a result